cats_frontend/src/app/modules/cats/pages/cats-list/cats-list.component.html

```diff
@@ -55,7 +55,7 @@
     <ng-template pTemplate="body" let-cat let-columns="columns" let-expanded="expanded">
         <tr>
             <td>
-                <button (click)="expandCat(cat)" type="button" pButton pRipple [pRowToggler]="cat" class="p-button-text p-button-rounded p-button-plain" [icon]="expanded ? 'pi pi-chevron-down' : 'pi pi-chevron-right'"></button>
```

Comment:
rename method expandCat to expand, the param cat already told us that's you are expanding a cat